packages/binding-http/test/http-client-test.ts

```diff
@@ -66,19 +68,22 @@ class TestHttpServer implements ProtocolServer {
         if (address !== undefined) {
             this.address = address;
         }
+        this.server = http.createServer((req, res) => {
+            this.checkRequest(req, res);
+        });
     }
 
-    public start(): Promise<void> {
-        return new Promise<void>((resolve, reject) => {
+    public async start(): Promise<void> {
+        return await new Promise<void>((resolve, reject) => {
```

Comment:
I would keep the previous code, I don't see the benefit of this inline promise.

Reply:
arguably, any method returning Promise shall be marked as async. I used this convention all across node-opcua for consistency, and I idiomatically applied the same convention here.

I will follow your convention in node-wot if you find appropriate to sometime have the async keyword and sometimes not , but I will also need clear rule to understand when to apply it and when not.

Reply:
Sorry I was referring to `await new Promise` at the end isn't it the same as simply returning ` new Promise`? 

About `async`, @danielpeintner already knows that I am fan of `async` keyword, so I would never stop you to add it 👍🏻 .